core/controllers/editor.py

```diff
@@ -102,7 +102,7 @@ def get(self, exploration_id):
                 iframe_restriction=None)
             return
 
-        (exploration, exploration_rights) = (
+        (_, exploration_rights) = (
             exp_services.get_exploration_and_exploration_rights_by_id(
```

Comment:
I think we should use rights_manager.get_exploration_rights, is there any issue with that? (as get_exploration_and_exploration_rights_by_id doing multi calls to datastore and we are using all values.)

Reply:
Yeah, okay

Reply:
Done